src/accordion/accordion.ts

```diff
@@ -33,9 +33,25 @@ let nextId = 0;
   directives: [NgbCollapse]
 })
 export class NgbPanel {
+  /**
+   *  A flag determining whether the panel is disabled or not.
+   *  When disabled, the panel cannot be toggled.
```

Comment:
If it is open and later becomes disabled, then one will not be able to close the panel. This language was intentional.